Upstream-Status: Inappropriate [embedded specific]

Currently default header is linux-libc-header, which leads the dismatch
with wrs kernel.

The different headers for same file cause trouble between kerenl module and
user level binary. E.g. segfault in v86d due to different
CONNECTOR_MAX_MSG_SIZE in connector.h from the one used by uvesafb.

This fix is work around to sync with wrs kernel headers, and should be removed
after switching to linux-libc-headers-wrs.

[BUGID #279] fixed by this.

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

Index: linux-2.6.34/include/linux/connector.h
===================================================================
--- linux-2.6.34.orig/include/linux/connector.h	2010-09-30 14:15:25.000000000 +0800
+++ linux-2.6.34/include/linux/connector.h	2010-09-30 14:15:43.000000000 +0800
@@ -48,7 +48,7 @@
 /*
  * Maximum connector's message size.
  */
-#define CONNECTOR_MAX_MSG_SIZE		16384
+#define CONNECTOR_MAX_MSG_SIZE		65536
 
 /*
  * idx and val are unique identifiers which 
